src/simulations/Simulations.jl

```diff
@@ -171,6 +179,7 @@ function LandSimulation(
     return LandSimulation(
         model,
         timestepper,
+        start_date,
```

Comment:
I added this since it was convenient to access from the simulation for the fluxnet default plotting. It's not stored otherwise, but seems like a useful number to be able to access